features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ApplicationSettings/EditApplication.jsx

```diff
@@ -124,8 +124,9 @@ function Edit(props) {
                 })
                 .catch((error) => {
                     const { response } = error;
-                    if (response.body) {
-                        throw response.body.description;
+                    if (response.body.code === 500) {
```

Comment:
catch clause could catch none-HTTP exceptions, in that case, error object will not have a body or code , so we need to do a sanity check here, We can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here

```javascript
if (response?.body?.code === 500)
```

Reply:
fixed with: https://github.com/wso2/carbon-apimgt/pull/9154/commits/e218dec7a0576687cb375829d06a5a5bb2b90708